Source/WebKitLegacy/mac/WebView/WebPreferencesDefaultValues.mm

```diff
@@ -106,29 +106,22 @@ bool defaultAllowContentSecurityPolicySourceStarToMatchAnyProtocol()
 
 bool defaultLoadDeferringEnabled()
 {
-    return !WebCore::MacApplication::isAdobeInstaller();
+    static bool isAdobeInstaller = WebCore::MacApplication::isAdobeInstaller();
+    return !isAdobeInstaller;
```

Comment:
This change is unnecessary. `WebCore::MacApplication::isAdobeInstaller` already caches its value.